util/stringutil/str_or_default.go

```diff
@@ -0,0 +1,9 @@
+package stringutil
+
+// StrOrDefault returns str or defaultStr when str is empty.
+func StrOrDefault(str, defaultStr string) string {
+	if str == "" {
+		return defaultStr
+	}
+	return str
+}
```

Comment:
Minor comment re filename: we haven't used `_` nor `-` in go file names before. Maybe rename to `strordefault.go`?